src/plots/cartesian/line_grid_defaults.js

```diff
@@ -32,18 +32,40 @@ module.exports = function handleLineGridDefaults(containerIn, containerOut, coer
         delete containerOut.linewidth;
     }
 
-    var gridColorDflt = colorMix(dfltColor, opts.bgColor, opts.blend || lightFraction).toRgbString();
+    var gridColorDflt = colorMix(dfltColor, opts.bgColor, opts.blend || colorAttrs.lightFraction).toRgbString();
     var gridColor = coerce2('gridcolor', gridColorDflt);
     var gridWidth = coerce2('gridwidth');
     var gridDash = coerce2('griddash');
-    var showGridLines = coerce('showgrid', opts.showGrid || !!gridColor || !!gridWidth || !!gridDash);
+    var showGridLines = coerce('showgrid', opts.showGrid ||
+        !!gridColor ||
+        !!gridWidth ||
+        !!gridDash
+    );
 
     if(!showGridLines) {
         delete containerOut.gridcolor;
         delete containerOut.gridwidth;
         delete containerOut.griddash;
     }
 
+    if(opts.hasMinor) {
+        var minorGridColorDflt = colorMix(containerOut.gridcolor, opts.bgColor).toRgbString();
```

Comment:
I think you're right that a 50/50 mix of these two is not muted enough for the default - maybe ~70% bgColor? But I do still think we should keep the width (here and for minor ticks) the same as the major widths. Given the way we round these widths (via `crispEdges`) the width change doesn't do anything unless you explicitly increase the major width, and in that case I think the expectation would be that all the lines increase together.